src/file.ts

```diff
@@ -33,6 +34,7 @@ import {Duplex, Writable, Readable} from 'stream';
 import * as streamEvents from 'stream-events';
 import * as through from 'through2';
 import * as xdgBasedir from 'xdg-basedir';
+import * as qs from 'query-string';
```

Comment:
Is there a reason `querystring.stringify` isn't good enough?
https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options

If there isn't - we should document the why :)

Reply:
I forgot this build-in module exists 😅

Reply:
refactored to use `querystring`. Thanks for catching that!